src/main/java/pvs/app/dto/GitlabIssueDTO.java

```diff
@@ -0,0 +1,44 @@
+package pvs.app.dto;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Data;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+import java.util.Date;
+
+@Data
+public class GitlabIssueDTO {
+    private String repoOwner;
+    private String repoName;
+    private Date createdAt;
+    private Date closedAt;
+
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Date createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public void setCreatedAt(JsonNode createdAt) {
+        DateTimeFormatter isoParser = ISODateTimeFormat.dateTimeNoMillis();
+        this.createdAt = isoParser.parseDateTime(createdAt.toString().replace("\"", "")).toDate();
+    }
+
+    public Date getClosedAt() {
+        return closedAt;
+    }
+
+    public void setClosedAt(Date closedAt) {
+        this.closedAt = closedAt;
+    }
+
+    public void setClosedAt(JsonNode closedAt) {
+        if (closedAt != null && closedAt.textValue() != null) {
+            DateTimeFormatter isoParser = ISODateTimeFormat.dateTimeNoMillis();
+            this.closedAt = isoParser.parseDateTime(closedAt.toString().replace("\"", "")).toDate();
+        }
+    }
```

Comment:
Suggest remove all getter and setter functions since you have already use the `lombok.Data` annotation in this class.

[nit] Although we don't need these methods anymore, but about the `setCreatedAt` and `setClosedAt` here, since they have overload methods, we should check if the overload function is used in other places. (Here the answer is not, so can be removed)

```suggestion
```